pkg/controller/pod_driver.go

```diff
@@ -135,8 +135,10 @@ func (p *PodDriver) checkAnnotations(ctx context.Context, pod *corev1.Pod) error
 	for k, target := range pod.Annotations {
 		if k == util.GetReferenceKey(target) {
 			targetUid := getPodUid(target)
+			// Only it is not in pod lists can be seen as deleted
+			// Add a 10s delay to avoid misjudgment of deletion due to cache reasons when the pod is just created.
```

Comment:
use consistent ending dot 😅

```suggestion
			// Add a 10s delay to avoid misjudgment of deletion due to cache reasons when the pod is just created
```